pkgs/dart_model/lib/src/scopes.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:async';
 import 'dart:typed_data';
 
+import '../dart_model.dart';
 import 'json_buffer/json_buffer_builder.dart';
 
 /// Scope for accumulating `dart_model` data for efficient transmission.
```

Comment:
Maybe expand this, something like: ...`dart_model` queries for efficient transmission, and responses for use in type queries.